Metadata-Matcher/config.py

```diff
@@ -0,0 +1,84 @@
+import os
+import logging
+
+# =============================================================================
+# CONFIGURATION
+#
+# You can override the configuration using environment variables
+# Example:
+#   $ export METADATA_FILE="/path/to/meta/file.xlsx"
+# =============================================================================
+
+# Paths
+METADATA_FILE = "/Users/nowke/Documents/diging/metadata/Cleaned Data 1900-2017.xlsx"
+TEXT_FILES_FOLDER = "/Users/nowke/Documents/diging/files"
+NOT_FOUND_FOLDER = "/Users/nowke/Documents/diging/not_found"
+FOUND_METADATA = "/Users/nowke/Documents/diging/found.xlsx"
```

Comment:
Let's change these to look in a default directory. https://stackoverflow.com/questions/6227590/finding-the-users-my-documents-path